packages/lune/src/builder/rolldown.ts

```diff
@@ -0,0 +1,32 @@
+import { SolidPlugin } from "./rolldown/solid";
+import { ShelterSolidResolver } from "./rolldown/resolver";
+import { LightningCSSPlugin } from "./rolldown/lightningcss";
+
+export async function createRolldownBuilder(entryPoint: string, outfile: string, minify: boolean, root: string) {
+  const { build } = await import("rolldown").catch(() => {
```

Comment:
I would much prefer we just depend on rolldown. This goes for lightningcss as well.

Reply:
What do you mean?